apps/server/src/modules/tool/school-external-tool/entity/SchoolExternalToolConfigurationStatusEntity.ts

```diff
@@ -0,0 +1,13 @@
+import { Entity } from '@mikro-orm/core';
+
+@Entity()
```

Comment:
You should define the table name here to be sure to match our required naming convention